src/EventListener/AccessControlListener.php

```diff
@@ -127,8 +145,9 @@ private function deleteSessions(Entity\Users $user)
     public static function getSubscribedEvents()
     {
         return [
-            StorageEvents::PRE_DELETE => 'onStorageEventPostSave',
-            StorageEvents::PRE_DELETE => 'onStorageEventPreDelete',
+            //\Bolt\Events\StorageEvents::PRE_DELETE => 'onStorageEventPostSave',
+            //\Bolt\Events\StorageEvents::PRE_DELETE => 'onStorageEventPreDelete',
```

Comment:
Did you comment these out instead of removing them on purpose?

Reply:
On purpose. Below you'll see this PR adds the listener. It hasn't been hooked up in a very very long time, so investigation of that isn't in scope for this one :smile_cat: 

A later PR can remove related logic if needed.